core/src/test/java/com/exadel/aem/toolkit/bundle/lists/models/ListEntryTest.java

```diff
@@ -34,13 +35,13 @@
 
     @Rule
     public AemContext context = new AemContext(ResourceResolverType.RESOURCERESOLVER_MOCK);
-    private ListEntry listEntry;
+    private ListItem listEntry;
```

Comment:
please rename the variable accordingly and move the test to 'internal' folder

Reply:
Done